paasta_tools/paasta_metastatus.py

```diff
@@ -218,8 +218,9 @@ def get_service_instance_stats(service: str, instance: str, cluster: str) -> Dic
 
     try:
         instance_type: str = validate_service_instance(service, instance, cluster, DEFAULT_SOA_DIR)
-    except Exception:
+    except Exception as e:
         log.error(f'Failed to get stats for service {service} instance {instance}:')
```

Comment:
Can you merge the following two `log.error()` calls.
It's better to output related log messages via a single `log.something` calls.